scripts/SANS/sans/test_helper/user_file_test_helper.py

```diff
@@ -77,6 +77,81 @@
                     "TUBECALIBFILE=TUBE_SANS2D_BOTH_31681_25Sept15.nxs"
                     )
 
+sample_user_file_2 = ("PRINT for changer\n"
```

Comment:
Rather than copy and paste this, could you use a function to generate these, with a parameter for the gravity value.
This would make it easier to tell the difference between the two values. and means you only need to update the values in one place if the format of the file changes.